### ISSUE TITLE
Capitalize network name

### ISSUE THREAD

**Issue by bjornwgnr** [paritytech/fether#506]
i.e.:
- `Synced foundation` -> `Synced Foundation`
- `Synced kovan` -> `Synced Kovan`

**Comment by ltfschoen** [comment]
Do we want it to say 'Synced Mainnet' instead of 'Synced Foundation'?

**Comment by Tbaut** [comment]
Yes, this is part of https://github.com/paritytech/fether/issues/486
